test/test_openai.py

```diff
@@ -97,6 +97,20 @@ def test_simple_chat_mode():
     assert out.completion_text.strip() == "4"
 
 
+def test_o1_mode():
+    lm = get_open_ai_lm(OpenAiModelNames.o1_preview_2024_09_12)
```

Comment:
do we want to do this with o1 mini? The point is the stress the codepath. It doesn't need to be the slowest/most costly model